lib/tooling/clang-format-tool.js

```diff
@@ -37,9 +37,9 @@ export class ClangFormatTool extends BaseTool {
         const options = compilationInfo.options;
         const dir = path.dirname(sourcefile);
 
-        const compileFlags = options.filter(option => (option !== sourcefile));
+        let compileFlags = options.filter(option => (option !== sourcefile));
```

Comment:
Super nitpick: Why this change? Just asking - any time something goes from `const` (albeit javascript's silly `const` notion) to `let` I have to ask :)

Reply:
because it gets reassigned below using concat

Reply:
Ah! Got it; and the options is now an array not a single string. We can use `compileFlags.push(*...this.tool.options)` but that's no clearer. Fine as-is! :)